packages/fxa-auth-server/fxa-oauth-server/lib/grant.js

```diff
@@ -133,9 +134,9 @@ module.exports.validateRequestedGrant = async function validateRequestedGrant(
 //
 // This function does *not* perform any authentication or validation, assuming that
 // the specified grant has been sufficiently vetted by calling code.
-module.exports.generateTokens = async function generateTokens(grant) {
+module.exports.generateTokens = async function generateTokens(grant, ppidSeed) {
   // We always generate an access_token.
-  const access = await exports.generateAccessToken(grant);
+  const access = await exports.generateAccessToken(grant, ppidSeed);
```

Comment:
Should we do a check here that for PPID enabled clients, that the grant should not contain the `profile` scope or any of its implicants? Otherwise the service provider or RP could present the access token to the profile server to get the user's profile info which contains the real uid and email address.